src/assets/i18n/en.json

```diff
@@ -1,44 +1,98 @@
 {
+  "ABBREVIATED_NAME_OF_THE_ORGANIZATION": "Abbreviated name of the organization",
   "ABOUT_THE_PORTAL": "ABOUT THE PORTAL",
+  "ACTUAL_ADDRESS": "Actual address",
   "ADD": "Add",
+  "ADD_ACHIEVEMENTS": "Add Achievements",
   "ADD_TO_FAVORITES": "Add to favorites",
   "ADDED_TO_FAVORITES": "Added to favorites",
-  "ADD_ACHIEVEMENTS": "Add Achievements",
   "ADMINISTRATION": "Administration",
   "ALL": "All",
+  "APPLICATIONS": "Applications",
+  "APPLY_AN_APPLICATION": "APPLY FOR THE WORKSHOP",
   "AVAILABLE": "Available",
-  "NOT_AVAILABLE": "Not available",
   "AWAITING_CONFIRMATION": "Awaiting confirmation",
   "BY_ALPHABET": "By Alphabet",
   "BY_PRICE_DECREASE": "By price decrease",
   "BY_PRICE_INCREASE": "By price increase",
   "BY_RATING": "By rating",
   "CANCEL": "Cancel",
   "CHANGE": "Change",
+  "CHANGE_EMAIL": "Change email",
+  "CHANGE_PASSWORD": "Change password",
   "CLEAR_FILTERS": "Clear Filters",
   "CLOSE_RECRUITMENT": "Closed Recruitment",
   "CONFIRM": "Confirm",
   "CONTACTS": "CONTACTS",
+  "DATE_OF_BIRTH_OF_THE_MANAGER": "Date of birth of the manager",
   "DAYS": {
-    "MONDAY": "Mon",
-    "TUESDAY": "Tue",
-    "WEDNESDAY": "Wed",
-    "THURSDAY": "Thu",
     "FRIDAY": "Fri",
+    "MONDAY": "Mon",
     "SATURDAY": "Sat",
-    "SUNDAY": "Sun"
+    "SUNDAY": "Sun",
+    "THURSDAY": "Thu",
+    "TUESDAY": "Tue",
+    "WEDNESDAY": "Wed"
   },
   "DIRECTIONS": "Categories",
+  "DIRECTOR_NAME": "Director's full name",
   "DISPLAY_ON_THE_PAGE": "Items per page:",
+  "DISTRICT_OF_THE_CITY": "District of the city",
   "DO_YOU_PROVIDE_EDUCATIONAL_SERVICES": "Do you provide educational services?",
+  "EDIT": "Edit",
+  "EDPNOU": "EDPNOU",
```

Comment:
EDRPOIPN : Unified State Register Code of Enterprises and Organizations of Ukraine